webpack.config.js

```diff
@@ -106,5 +107,15 @@ module.exports = {
       });
       return middlewares;
     }
-  }
+  },
+  optimization: {
+    minimize: true,
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          keep_classnames: true
```

Comment:
I wonder if this suggests we are still not quite bundling the way we want. I hate to say it, but maybe we need 3 exports. the original source(ESM) a commonjs version of the original source, and then the minified bundle for use in web browsers? Not suggesting we change it right now, but something to think about